ibm_cloud_sdk_core/base_service.py

```diff
@@ -41,6 +42,8 @@
 # import http.client as http_client
 # http_client.HTTPConnection.debuglevel = 1
 
+#pylint: disable=too-many-instance-attributes
+#pylint: disable=too-many-locals
```

Comment:
We have 8/7 instance attributes and 17/15 locals with the newly added `gzip_compression` variables, so I had to disable these lint checks for this class.